man/man8/zfs.8

```diff
@@ -3304,7 +3304,7 @@ Displays all ZFS file systems currently mounted.
 .Op Fl o Ar options
 .Fl a | Ar filesystem
 .Xc
-Mounts ZFS file systems.
+Mount ZFS filesystem if a non-legacy mountpoint is set.
```

Comment:
I think we could be even more specific by saying, `Mount ZFS filesystem on the directory specified by the 'mountpoint' property.  However, if the mountpoint property is 'legacy', the mount request is ignored and the filesystem is not mounted.`